addons/account/tests/test_fiscal_position.py

```diff
@@ -14,7 +14,7 @@ def setUp(self):
 
     def test_fiscal_position(self):
         cr, uid = self.cr, self.uid
-        country_id = self.res_country_model.search(cr, uid, [('name', '=', 'France')])[0]
+        country_id = self.res_country_model.search(cr, uid, [('name', '=', 'Belgium')])[0]
```

Comment:
Actually In test case Belgium vat number was used and country was France so thats why i had Fixed test case,anyway i have Improved it with using 'xmlid_to_res_id'.